api/envoy/config/filter/http/local_rate_limit/v2/local_rate_limit.proto

```diff
@@ -0,0 +1,57 @@
+syntax = "proto3";
+
+package envoy.config.filter.http.local_rate_limit.v2;
+
+import "envoy/api/v2/core/base.proto";
+import "envoy/type/http_status.proto";
+import "envoy/type/token_bucket.proto";
+
+import "udpa/annotations/migrate.proto";
+import "udpa/annotations/status.proto";
+import "validate/validate.proto";
+
+option java_package = "io.envoyproxy.envoy.config.filter.http.local_rate_limit.v2";
+option java_outer_classname = "LocalRateLimitProto";
+option java_multiple_files = true;
+option (udpa.annotations.file_migrate).move_to_package =
+    "envoy.extensions.filters.http.local_ratelimit.v3";
+option (udpa.annotations.file_status).package_version_status = FROZEN;
+
+// [#protodoc-title: Local Rate limit]
+// Local Rate limit :ref:`configuration overview <config_http_filters_local_rate_limit>`.
+// [#extension: envoy.filters.http.local_ratelimit]
+
+message LocalRateLimit {
+  // This field allows to send a HTTP response status code to the downstream client other
+  // than 429 (TooManyRequests) when the request has been rate limited.
+  type.HttpStatus status = 1;
+
```

Comment:
I have some questions and discussed by https://github.com/envoyproxy/envoy/issues/10813.